app/src/test/java/waiter/RouterTest.java

```diff
@@ -10,59 +10,59 @@ public class RouterTest {
     void getSimpleGetReturnsOKRouteWithoutBody() {
         Request request = new Request("GET","/simple_get", "HTTP/1.1");
 
-        Route route = new Router().getRoute(request);
+        Response Response = new Router().getRequestedRoute(request);
 
-        assertEquals("HTTP/1.1", route.getProtocol());
-        assertEquals("200 OK", route.getStatus());
-        assertEquals("Content-Length: 0", route.getHeaders());
-        assertEquals("", route.getBody());
+        assertEquals("HTTP/1.1", Response.getProtocol());
+        assertEquals("200 OK", Response.getStatus());
+        assertEquals("Content-Length: 0", Response.getHeaders());
+        assertEquals("", Response.getBody());
     }
 
     @Test
     void getSimpleGetWithBodyReturnsOKRouteWithBody() {
         Request request = new Request("GET","/simple_get_with_body", "HTTP/1.1");
 
-        Route route = new Router().getRoute(request);
+        Response Response = new Router().getRequestedRoute(request);
 
-        assertEquals("HTTP/1.1", route.getProtocol());
-        assertEquals("200 OK", route.getStatus());
-        assertEquals("Content-Length: 11", route.getHeaders());
-        assertEquals("Hello world", route.getBody());
+        assertEquals("HTTP/1.1", Response.getProtocol());
+        assertEquals("200 OK", Response.getStatus());
+        assertEquals("Content-Length: 11", Response.getHeaders());
+        assertEquals("Hello world", Response.getBody());
     }
 
     @Test
     void headSimpleGetReturnsOKRouteWithoutBody() {
         Request request = new Request("HEAD","/simple_get", "HTTP/1.1");
 
-        Route route = new Router().getRoute(request);
+        Response Response = new Router().getRequestedRoute(request);
 
-        assertEquals("HTTP/1.1", route.getProtocol());
-        assertEquals("200 OK", route.getStatus());
-        assertEquals("Content-Length: 0", route.getHeaders());
-        assertEquals("", route.getBody());
+        assertEquals("HTTP/1.1", Response.getProtocol());
+        assertEquals("200 OK", Response.getStatus());
+        assertEquals("Content-Length: 0", Response.getHeaders());
+        assertEquals("", Response.getBody());
     }
 
     @Test
     void NoMethodOrUrlFoundReturnsInternalServerErrorRouteWithoutBody() {
```

Comment:
I don't think this description is accurate since we're returning 404 instead of internal server error.

Reply:
Good catch, changing this now, thank you! 😄